src/sentry/static/sentry/app/views/settings/components/dataPrivacyRulesPanel/dataPrivacyRulesPanel.tsx

```diff
@@ -245,14 +245,14 @@ class DataPrivacyRulesPanel extends React.Component<Props, State> {
         <Panel>
           <StyledPanelHeader>{t('Data Privacy Rules')}</StyledPanelHeader>
           <PanelAlert type="info">
-            {additionalContext}
-            {tct('To learn more about datascubbing, [linkToDocs].', {
+            {additionalContext}{' '}
```

Comment:
This adds a space character to separate the two parts without relying on `additionalContext` having trailing whitespace.